### DISCUSSION TITLE
trouble installing lightkurve via Anaconda

### DISCUSSION THREAD

**Discussion by eddiemorris135** [lightkurve/lightkurve#1273] [General]
Hello All,

Currently I am having trouble installing lightkurve via Anaconda https://anaconda.org/conda-forge/lightkurve. I have tried the first 3 but each came with different errors. Is there a more update version I could use? I use the Spyder IDE via Anaconda.

Thanks
Ed

**Comment by eddiemorris135** [comment]
Solution:
1)  uninstalled Anaconda
2) re-install with admin rights
3) selected the first option from that link and run anaconda prompt with admin rights.
Working so far.

**Comment by orionlee** [comment]
I'm glad you get it work!

Having said that, installing lightkurve does not require Anaconda to be installed / run with admin rights in general. It's possible your machine has some specific setup that requires so.

I just tested on my machine, and I could install lightkurve without admin rights.

My setup is:
- Windows 11
- Anaconda 2019.07, conda upgraded to v 22.11
- `conda-forge` channel has the priority.

**Comment by eddiemorris135** [comment]
Hi,

Thanks for the tips! I should get more familiar with my machine!
